### DISCUSSION TITLE
Lampade Smart + Interruttore Smart

### DISCUSSION THREAD

**Discussion by Cusu95** [indomus/forum#1445] [GENERALE - Discussioni generiche]
Ciao a tutti,
al momento ho dei punti luce dove ho delle lampade smart. Il problema è che spesso, soprattutto quando va via internet, il controllo diventa un inferno.
Al momento sto gestendo le luci con dei pulsanti zigbee di Ikea, funzionano abbastanza bene anche se a volte hanno un po' di latenza.

La mia domanda è: è possibile gestire delle lampadine smart con degli interruttori smart, senza farle andare offline, per mantenere il controllo dell'intensità e del colore?

In caso negativo, quale sarebbe la soluzione migliore per la gestione dei punti luce? (possibilmente con lampadine rgb)

Grazie mille a tutti quelli che vorrano rispondere

**Comment by snakuzzo** [comment]
Le soluzioni sono decine. Devi semplicemente assicurarti che i device che vuoi pilotare non dipendano da servizi in cloud

**Reply by Cusu95** [reply]
Al momento utilizzo ZHA (sia per le lampade che per i telecomandi Ikea), proprio perché con i dispositivi WIFI avevo sempre il problema della disconnessione internet

**Reply by snakuzzo** [reply]
allora a posto così...anche se la connessione ad internet non va, funzionerà tutto lo stesso

**Reply by Cusu95** [reply]
> 

prima con deconz avevo qualche problema, anche se non sono riuscito a capire il perché. Adesso con ZHA è più reattivo, anche se ogni tanto i pulsanti ikea perdono qualche comando. 
Ci sarebbe una soluzione più "pulita"? Per non avere i pulsanti ikea attaccati a muro, qualcosa come bticino o vimar, che funzioni anche se homeassistant andasse giù per qualsivoglia motivo.

So che sto chiedendo troppo, scusami, ma ti ringrazio già tanto per le tue risposte

**Reply by snakuzzo** [reply]
Se utilizzi i pulsanti IKEA a batteria...può starci che fallisca qualche volta (ma andrebbe comunque indagato) perché vanno in deep sleep, ma comunque, avendolo con ZHA, è del tutto indipendente sia dalla WIFI che dalla connettività verso internet. Il device comunica con il coordinator che (molto probabilmente) avrai collegato alla USB del server su cui gira HA con l'integrazione ZHA. Quindi...a livello di rete potresti avere tutti i disservizi che ti pare, ma il device dovrebbe sempre funzionare. 
Utilizzando dispositivi alimentati, questo problema in teoria non dovresti averlo, ma stiamo parlando di cose diverse. Quelli IKEA, come detto prima, solitamente sono pulsanti a batteria. Quelli vimar o bticino sono solitamente i frutti da inserire nelle scatole per pilotare luci, tapparelle, etc e sono alimentati a 220V (non vanno in deep sleep).
